neurodsp/filt/utils.py

```diff
@@ -178,7 +185,15 @@ def compute_transition_band(f_db, db, low=-20, high=-3):
     # This gets the indices of transitions to the values in searched for range
     inds = np.where(np.diff(np.logical_and(db > low, db < high)))[0]
     # This steps through the indices, in pairs, selecting from the vector to select from
-    transition_band = np.max([(b - a) for a, b in zip(f_db[inds[0::2]], f_db[inds[1::2]])])
+    transition_pairs = [(a, b) for a, b in zip(f_db[inds[0::2]], f_db[inds[1::2]])]
```

Comment:
the previous description here (line 187) is... not great. 
Since you've updated and extended this section, could you add some comments detailing more about what it's doing, and what it's extracting here?